src/registered.js

```diff
@@ -71,7 +71,15 @@ module.exports = async function (request, h) {
     const data = await res.json();
 
     if (!res.ok) {
-      throw `errore imprevisto nel salvataggio, riprovare più tardi (${data?.title}, ${data?.detail})`;
+      let errorMsg = '';
+      if (data.validationErrors) {
+        for (const error of data.validationErrors) {
+          errorMsg += `Valore non valido: <b>${error.value}</b> per il field: <b>${error.field}</b> con la regola: <b>${error.rule}</b></br>`;
+        }
+        throw errorMsg;
+      }
+
+      throw data.detail;
```

Comment:
```suggestion
       throw data.validationErrors?.map(e => `
         Valore non valido: <strong>${error.value}</strong> per il campo: <strong>${error.field}</strong>
         con la regola: <strong>${error.rule}</strong><br>
       `).join('') || data.detail
```

Reply:
Yes, thank you. Much better.
I haven't played in javascript for a while 🙁